src/main/java/Player.java

```diff
@@ -0,0 +1,56 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Player {
+    private int settlements;
+    private int cities;
+    private int roads;
+    private int score;
+    private List<DevelopmentCard> developmentCards;
+    private Wallet wallet;
+    int numRoads;
+
```

Comment:
Should there be an arrayList of cities, settlements, and roads to keep track of their location or should that be part of Board?